src/modules/m_spanningtree/main.cpp

```diff
@@ -210,7 +210,7 @@ void ModuleSpanningTree::ConnectServer(Link* x, Autoconnect* y)
 	else
 	{
 		in_addr n;
-		if (inet_aton(x->IPAddr.c_str(),&n) < 1)
+		if (inet_pton(AF_INET, x->IPAddr.c_str(),&n) < 1)
```

Comment:
The first arm of this if statement does the same as this only with an in6_addr and AF_INET6.